mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -804,6 +816,21 @@ msgid "welcome_location_town"
 msgstr "Welcome to ${{map_name}} Town: ${{map_desc}}\n"
 "North: ${{north}} / South: ${{south}} / West: ${{west}} / East: ${{east}}"
 
+msgid "here_to_north"
+msgstr "From ${{map_name}} to ${{north}}"
+
+msgid "here_to_south"
+msgstr "From ${{map_name}} to ${{south}}"
+
+msgid "here_to_west"
+msgstr "From ${{map_name}} to ${{west}}"
+
+msgid "here_to_east"
+msgstr "From ${{map_name}} to ${{east}}"
+
+msgid "alert_route"
+msgstr "Rehabilitation will begin as soon as possible. \n Please stop nagging us."
```

Comment:
You are about this one?

Reply:
Originally:
`msgstr "Route 3: Rehabilitation will begin as soon as possible. \n Please stop nagging us."`
Now the sign shows "From XXX to Route 3", then it appears another dialogue with "alert_route"
Yeah, probably wrong position, I'm going to move it close the route3 dialogues.